### PR TITLE
Adds 3 new toys to the prize pool not removing all of science edition.

### PR DESCRIPTION
**Pull request by itseasytosee**
## About The Pull Request
Adds three more toys to the pool for arcade cabinets.
For people who already saw this, there is no new content.

Trick Blindfold A clever trick that allows you to cheat at games like pin the security baton on the clown, or pinata games

Squeaky Brain A toy that looks like a brain that makes a squishing sound. (uses old brain sprite)

Last but not least (personal favorite)

Broken Radio
An old radio that can't pick up anything but static.
It plays "static" which is radiation ticking for about twelve seconds when used (with a reasonable cooldown)
Used a remodeled Geiger counter sprite.

In addition, it does not add a million commits
## Changelog
:cl: itseasytosee
add: Three more toys to find in your local arcade cabinet! Squeaky brain, trick blindfold, and a broken radio. Collect em all!
/:cl:

### CODE REVIEW

File: code/game/objects/items/toys.dm [hunk 1]
```diff
@@ -1489,3 +1491,34 @@
 	icon_state = "shell[rand(1,3)]"
 	color = pickweight(possible_colors)
 	setDir(pick(GLOB.cardinals))
+
+/obj/item/toy/brokenradio
+	name = "broken radio"
+	desc = "An old radio that produces nothing but static when turned on."
+	icon = 'icons/obj/toy.dmi'
+	icon_state = "broken_radio"
```
Comment (ATH1909) [inline code/game/objects/items/toys.dm]: you could also use the nuke op radio uplink sprite

Comment (itseasytosee) [reply code/game/objects/items/toys.dm]: I think the Geiger counter Esq sprite works well as it keeps with the theme. On donut, there is already an "old radio" that uses the sprite

Comment (trollbreeder) [reply code/game/objects/items/toys.dm]: Kilo uses the old radio too, in the bar and the atrium

**Comment by Dorsisdwarf** [comment]
Seems weird that the trick blindfold is openly called a trick blindfold, so everyone can instantly see that you're cheating.

**Comment by trollbreeder** [comment]
> Seems weird that the trick blindfold is openly called a trick blindfold, so everyone can instantly see that you're cheating.

Should only reveal it's a trick blindfold on examine.

**Comment by wesoda25** [comment]
👎 old pr was better, we don’t need science

**Comment by itseasytosee** [comment]
>Should only reveal it's a trick blindfold on examine.

Sounds good

**Comment by itseasytosee** [comment]
The trick blindfold also has little purple bits on the strings but you can't see it on the sprite while wearing so if you had it in your hand it would be obvious.

**Review by kriskog — APPROVED** [review]

**Pull request merged**